docker-compose.yml

```diff
@@ -19,7 +21,7 @@ consul:
 # Scale this tier and each additional container/instance will automatically self-configure as a member of the cluster
 #
 couchbase:
-    image: misterbisson/triton-couchbase
+    image: misterbisson/triton-couchbase:enterprise-3.0.3-r2
```

Comment:
This image is pushed. Changes are in https://github.com/misterbisson/triton-couchbase/pull/2